yarn-project/end-to-end/src/bench/client_flows/account_deployments.test.ts

```diff
@@ -81,9 +83,11 @@ describe('Deployment benchmark', () => {
               1, // Kernel tail
           );
 
-          // Ensure we paid a fee
-          const tx = await deploymentInteraction.send(options).wait();
-          expect(tx.transactionFee!).toBeGreaterThan(0n);
+          if (process.env.SANITY_CHECKS) {
```

Comment:
Sanity checks are useful if things look odd or during development but they do slow down benchmarking significantly, since they used the fully proven PXE